### PR TITLE
Cleanup and documentation

### PR DESCRIPTION
**Pull request by aperrin66**
I was adapting the TOPVOYS configuration files to the new include option and I noticed some things to clean up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,8 +91,11 @@ The properties which are common to every harvester are:
 
 The rest depends on the harvester and will be detailed in each harvester's documentation.
 
-- **time_range** (optional): a two-elements list containing two date strings which define a time
-  range to which the crawler will be limited.
+- **time_range** (optional): for harvesters which inherit from `WebDirectoryHarvester`. A
+  two-elements list containing two date strings which define a time range to which the crawler will
+  be limited.
+- **include** (optional): for harvesters which inherit from `WebDirectoryHarvester`. A regular
+  expression matching the path of the files to include in the remote repository.
```
Comment (opsdep) [inline README.md]: please, complete it like this:
`...matching the path of the files to include in the remote repository(or the path of the files/folders to include in local addresses)`

File: geospaas_harvesting/ingesters.py [hunk 2]
```diff
@@ -643,10 +642,8 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
                         for dataset_param in json_loads_result
                     ]
             else:
-                self.LOGGER.error(
-                    "'dataset_parameters' section of metadata is not a json-dumped python list",
-                    exc_info=True)
                 raise TypeError(
-                    "'dataset_parameters' section of metadata is not a json-dumped python list")
+                    f"Can't ingest '{dataset_info}': the 'dataset_parameters' section of the "
+                    "metadata returned by nansat is not a list")
```
Comment (opsdep) [inline geospaas_harvesting/ingesters.py]: you forget `json-dumped`

File: tests/test_ingesters.py [hunk 3]
```diff
@@ -952,7 +952,8 @@ def test_exception_handling_of_bad_development_of_mappers(self):
             normalized_attributes = ingester._get_normalized_attributes('')
         self.assertEqual(
             err.exception.args[0],
-            "'dataset_parameters' section of metadata is not a json-dumped python list")
+            "Can't ingest '': the 'dataset_parameters' section of the metadata returned by nansat "
+            "is not a list")
```
Comment (opsdep) [inline tests/test_ingesters.py]: you forget `json-dumped`

**Review by opsdep — COMMENTED** [review]
noted inline

File: README.md (hunk 1 above)
Comment (aperrin66) [reply README.md]: I replaced remote with "harvested". That way it covers the local case and it's less verbose.

File: geospaas_harvesting/ingesters.py (hunk 2 above)
Comment (aperrin66) [reply geospaas_harvesting/ingesters.py]: done

File: tests/test_ingesters.py (hunk 3 above)
Comment (aperrin66) [reply tests/test_ingesters.py]: done

**Review by opsdep — APPROVED** [review]

**Pull request merged**